iam/__tests__/clearTextGithubOrg.test.ts

```diff
@@ -73,15 +75,16 @@ beforeEach(() => {
 describe("Attempt verification", function () {
   const pii = `${org}#${validGithubUserResponse.data.id}`
   it("handles valid verification attempt", async () => {
```

Comment:
I would add one more test:
```
it("handles valid verification attempt when no requestedClient is provided", ...
```
and ensure that the correct credentials are used in that case  (these credentials `process.env.GITHUB_*` )

Reply:
Good call :) I added an additional test to cover this